### DISCUSSION TITLE
USING python elastalert/elastalert.py to run this project,but got a error!

### DISCUSSION THREAD

**Discussion by bmbbms** [jertel/elastalert2#417] [Q&A] [answered]
Traceback (most recent call last):
  File "elastalert/elastalert.py", line 32, in <module>
    from elastalert import kibana
  File "/data/elastalert2/elastalert2-master/elastalert/elastalert.py", line 32, in <module>
    from elastalert import kibana
ImportError: cannot import name 'kibana'

# how can i work normally by this way

**Comment by nsano-rururu** [comment, marked as answer]
- Did you read the documentation?
- docker or direct installation?
- What command did you execute?
  - Did you really install it?
- What version of elasitcsearch?

**Reply by bmbbms** [reply]
Thanks for your reply.Actually, i install the elastalert2 on my machine. **i download the source code,then move to root  directory of this project,And use the above command "python  elastalert/elastalert.py",finally meeting the above mistakes.** if i copy the elastalert/elastalert.py to another workplace,and set a new filename ,it can work well.but it is not a good way for me debug this project.

**Reply by nsano-rururu** [reply]
What is the python version? .. In other words, you didn't install it, right?

**Reply by nsano-rururu** [reply]
What is the purpose? .. Function addition? .. Do you want to use it?

**Reply by nsano-rururu** [reply]
Didn't you read it?
https://elastalert2.readthedocs.io/en/latest/running_elastalert.html#as-a-python-package
